ngraph/frontend/paddlepaddle/src/pdpd_utils.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "frontend_manager/frontend_exceptions.hpp"
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+#ifdef _WIN32
+            const char PATH_SEPARATOR = '\\';
+#if defined(ENABLE_UNICODE_PATH_SUPPORT)
+            const wchar_t WPATH_SEPARATOR = L'\\';
+#endif
+#else
+            const char PATH_SEPARATOR = '/';
+#endif
+
+            template <typename T>
+            inline std::basic_string<T> get_path_sep()
+            {
+                return std::basic_string<T>{PATH_SEPARATOR};
+            }
+
+#if defined(ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
+            template <>
+            inline std::basic_string<wchar_t> get_path_sep()
+            {
+                return std::basic_string<wchar_t>{WPATH_SEPARATOR};
+            }
+#endif
+
+            template <typename T>
+            bool endsWith(const std::basic_string<T>& str, const std::basic_string<T>& suffix)
```

Comment:
It looks like a common util. should each frontend implement it?

Reply:
You mean to create static util library?

Reply:
Something like that. Moreover if I remember right we have the same methods in the Inference Engine

Reply:
How can I access methods from IE? I thought we don't link IE to ngraph

Reply:
In general it looks like it should be a common library. Could you add TODO to move such utility methods to common library and create a task?